ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-multicast/ovn-sdn-migration-feature-migration-validate-multicast-commands.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+validate_namespace_multicast_annotation () {
+  kubectl get namespace test-migration -o json | jq .metadata.annotations | grep multicast-enabled | tee current_config
+  echo "$EXPECT_NAMESPACE_MULTICAST" | tee expected_config
+  if diff <(jq -S . current_config) <(jq -S . expected_config); then
```

Comment:
I think this is giving an error:

++ jq -S . expected_config
parse error: Expected string key before ':' at line 1, column 32
parse error: Expected string key before ':' at line 1, column 34

even if the error is meaningless and doesn't affect anything, it's scary to look at. is there a way
to not have it?

Reply:
Honestly, I'm not sure. It _does_ seem to be a meaningless error, but I agree it is scary to look at. Leaving this unresolved because it's a lower priority thing to fix, but I still want to come back and fix this.

Reply:
I fixed this in my most recent commit. I just formatted the string to follow the style used in the rollback/egressFirewall e2e's.

Reply:
@jluhrsen I think this is actually causing failures/issues that are larger than just a scary error message. I'm going to revert to the previous approach for now. I think it should be okay because the tests themselves were passing fine then.